.changeset/wicked-dingos-drop.md

```diff
@@ -0,0 +1,8 @@
+---
+"@nomicfoundation/slang": minor
```

Comment:
I suggest using a `patch`, since this a bug fix, and doesn't break the public API.

```suggestion
"@nomicfoundation/slang": patch
```